### PR TITLE
OOP school library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
In this PR, I have created classes for Person, Student and Tecaher.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @id = Random.rand(1...100)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services
+    return true if id is_of_age? || @parent_permission
```
Comment (mikethreels) [inline person.rb]: - [x] it seems in this line you forgot to remove the `id`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Boaz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (Aduda-Boaz) [reply person.rb]: Thank you for this observation. I have resolved it now.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Aduda-Boaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**